lib/board.rb

```diff
@@ -0,0 +1,63 @@
+class Board
+  attr_accessor :cells, :turn_count, :WIN_COMBINATIONS
+  def initialize
+    @cells = cells
+    reset!
+  end
+
+  def cells
+    @cells
+  end
+
+  def reset!
+    self.cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    @cells[input.to_i - 1]
+  end
+
+  def full?
+    @cells.none? do |space|
+      if space == " "
+        true
+      else
+        false
+      end #ends if/else
+    end #ends block code
+  end #ends full? method
+
+  def turn_count
+    @cells.count {|c| c == "X" || c == "O"}
+  end
+
+  def taken?(move)
+    @cells[move.to_i - 1] == "X" || @cells[move.to_i - 1] == "O"
+  end
+
+  def valid_move?(move)
+    move_taken = move.to_i
```

Comment:
Use 'and' operator (`&&`) to combine multiple logical expressions and utilize their implicit return value.
```
move.to_i.between?(1,9) && !taken(input)
```